docs/Makefile

```diff
@@ -51,7 +51,7 @@ help:
 clean:
 	rm -rf $(BUILDDIR)/*
 	rm -rf auto_examples
-	rm -rf autosummary
+	rm -rf modules/autosummary
```

Comment:
Why does this change?

Reply:
Because the directory autosummary is generated inside modules directory.

Reply:
Ok, so this corrects a previous bug, in that case, because you did not change the generation.